frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+import * as React from 'react';
+import { Row } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { useLeaflet } from 'react-leaflet';
+import { MAX_ZOOM } from 'constants/strings';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+const LinkMenu = styled(Row)`
+  background-color: #f2f2f2;
+  height: 35px;
+  width: 341px;
+  margin-left: -10px;
+  font-size: 14px;
+  font-family: ProximaNova, sans-serif;
```

Comment:
We use BC-Sans everywhere else, so this concerns me. Was another font requested in the design?

Reply:
This one was used in the design, not sure if it matters? @bbjornholt ?

Reply:
I think that's my mistake. This can/should be BCSans. thanks!